phoenix-load-balancer/src/main/java/org/apache/phoenix/loadbalancer/service/LeastLoadStrategy.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.phoenix.loadbalancer.service;
+
+import org.apache.curator.x.discovery.ProviderStrategy;
+import org.apache.curator.x.discovery.ServiceInstance;
+import org.apache.curator.x.discovery.details.InstanceProvider;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Created by rshrivastava on 3/4/17.
```

Comment:
Strike this and others, please. Consider adding some meaningful class-level javadoc :)